lib/examples/network/model/user.dart

```diff
@@ -13,10 +13,10 @@ class User {
   Map<String, dynamic> toJson() {
     final Map<String, dynamic> data = <String, dynamic>{};
     if (this.data != null) {
-      data['data'] = this.data!.toJson();
+      data['data'] = this.data?.toJson();
```

Comment:
why you had to change this? itdoesn't make sense since you already did a check for in case it was `null`